src/shared/components/FileUpload/FileUpload.tsx

```diff
@@ -50,23 +51,30 @@ const FileUpload: FunctionComponent<FileUploadProps> = ({
 	const [isProcessing, setIsProcessing] = useState<boolean>(false);
 
 	const uploadSelectedFile = async (file: File | null) => {
-		if (file) {
-			if (allowedTypes.includes(file.type)) {
-				setIsProcessing(true);
-				const url = await uploadFile(file, assetType, ownerId);
-				onChange(url);
-				setIsProcessing(false);
-			} else {
-				const allowedExtensions = allowedTypes
-					.map(type => type.split('/').pop() || type)
-					.join(', ');
-				toastService.danger(
-					t('Het geselecteerde bestand is niet toegelaten, ({{allowedExtensions}})', {
-						allowedExtensions,
-					})
-				);
+		try {
+			if (file) {
+				if (allowedTypes.includes(file.type)) {
+					setIsProcessing(true);
+					const url = await uploadFile(file, assetType, ownerId);
+					onChange(url);
+				} else {
+					const allowedExtensions = allowedTypes
+						.map(type => type.split('/').pop() || type)
+						.join(', ');
+					toastService.danger(
```

Comment:
Toasts zijn niet goed zichtbaar in beheer (dark op dark achtergrond)

Reply:
inderdaad, maar ik ben er nog niet helemaal uit hoe we dit gaan oplossen, aangezien deze component ook gebruikt wordt om een bundel cover image te uploaden. De mooiste oplossing zal wss zijn dat de toastservice default naar dark/light als er geen expliciete dark/light property is meegegeven en dan kijkt naar de url en light toont voor het admin dashboard?